app/controllers/questions_controller.rb

```diff
@@ -52,7 +52,8 @@ def new
   def edit; end
 
   def create
-    @question = current_user.questions.new(question_params)
+    @question = Question.new(question_params)
+    @question.user = current_user unless admin_login? || current_user.mentor?
```

Comment:
```suggestion
    @question.user = current_user if !admin_or_mentor_login? 
```

unlessとorが両方ある場合に理解しずらいので避けるのがいいと思います。

Reply:
ありがとうございます🙏提案いただいた書き方に修正しました。
また、`app/views/questions/_form.html.slim` の、「フォーム表示に関する条件分岐（55行目）」も同じメソッドを使うように修正しました。